config.go

```diff
@@ -105,7 +105,6 @@ type config struct {
 	AdminIPs           []string `long:"adminips" description:"Expected admin host"`
 	AdminUserIDs       []string `long:"adminuserids" description:"User IDs of users who are allowed to access administrative functions."`
 	MinServers         int      `long:"minservers" description:"Minimum number of wallets connected needed to avoid errors"`
-	EnableStakepoold   bool     `long:"enablestakepoold" description:"Enable communication with stakepoold"`
```

Comment:
Since unrecognized parameters throw errors, I'm generally in favor of deprecating before removing.  That is make all the changes in this PR, except leave this line and change the description to say that something like "(deprecated. stakepoold is required)".  Then config.go can just print a warning if `enablestakepoold` is set that the user should remove the option since it's deprecated.

Reply:
OK! I'll do that. Thank you.

Reply:
Please also rebase and resolve the server.go conflict.  Lemme know if you need git workflow tips.